src/main/java/com/coveo/saml/SamlClient.java

```diff
@@ -245,7 +250,7 @@ public String getSamlRequest() throws SamlException {
     logger.trace("Issuing SAML request: " + stringWriter.toString());
 
     try {
-      return Base64.encodeBytes(stringWriter.toString().getBytes("UTF-8"));
+      return Base64.getEncoder().encodeToString(stringWriter.toString().getBytes("UTF-8"));
```

Comment:
@malaporte : FYI after testing this code for a while we found one case where Java's Base64 class was failing to decode some assertions. I didn't have time to dig more into it but switching to [Apache's Base64](https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Base64.html) fixed it for us (this applies to other places in this PR as well).